service/create/service.go

```diff
@@ -954,6 +954,7 @@ func (s *Service) processCluster(cluster awstpr.CustomObject) error {
 		Name:        cluster.Spec.Cluster.Kubernetes.API.Domain,
 		Clients:     clients,
 		Cluster:     cluster,
+		IdleTimeoutSeconds: cluster.Spec.AWS.ELB.IdleTimeoutSeconds.API,
```

Comment:
What happens with old TPOs for which `cluster.Spec.AWS.ELB` is nil? I'm worried about panics at this point.

Reply:
Good point, I'll do something about this

Reply:
Hi Federico, an empty struct cannot be nil, so `cluster.Spec.AWS.ELB` and `cluster.Spec.AWS.ELB.IdleTimeoutSeconds` will both be empty, but not nil.  Because the `API`, `Etcd`, and `Ingress` field are ints, they will be zero by default.  If a zero or negative value is passed as Input, it will be increased to the default value (60).  Checking for the maximum value will be done in "Validation" since it can be done while parsing, and does not need to be a "runtime" error.

Reply:
Indeed, sorry I was missing the changes in awstpr, you are totally right about zero values given the current types.